File: lab2-semester1/3.cpp
//C++  Program to Calculate Area of Equilateral Triangle

#include<iostream>
#include <cmath>
using namespace std;

int main(){
    float a, area;

    cout<<"Enter lenght of side: ";
    cin>>a;
   
    area=(sqrt(3)/4)*a*a;
    
      cout<<"\nArea of Equilateral Triangle = "<<area<<endl;
}

File: lab2-semester1/3.go
//Program to Calculate Area of Equilateral Triangle

package main

import (
    "fmt"
    "math"
    )

func main() {
    var a, area float64
    
    fmt.Print("Input lenght of side: ")
    fmt.Scanln(&a)//input from user
    
    area=(math.Sqrt(3)/4)*a*a //formula to calculate area of triangle
    
    fmt.Println("Area: ", area)
}
